diagnostic_feedback/tests/integration/base_test.py

```diff
@@ -0,0 +1,183 @@
+from selenium.webdriver.support.ui import WebDriverWait
+from xblock.fields import String
+from xblockutils.base_test import SeleniumXBlockTest
+from xblockutils.resources import ResourceLoader
+import time
+# Studio adds a url_name property to each XBlock but Workbench doesn't.
+# Since we rely on it, we need to mock url_name support so it can be set via XML and
+# accessed like a normal field.
+from diagnostic_feedback.quiz import QuizBlock
+QuizBlock.url_name = String()
+
+loader = ResourceLoader(__name__)
+
+
+class DiagnosticFeedbackBaseTest(SeleniumXBlockTest):
+    """
+    The new base class for integration tests.
```

Comment:
@jareerahsan Nit: In the future it won't matter too much that this class was once new :) You could shorten this to "Base class for integration tests."